### ISSUE TITLE
zeppelin introduction 

### ISSUE THREAD

**Issue by andrewpsp** [andrewpsp/the_kube_way#5]
I was looking into this tool, it seems like it will be the glue as far Point if contact for any other applications. I can create a micro service with this app.

**Comment by markjacksonfishing** [comment]
Yup, the notebook allows for mS creation. The only downfall is the git part of zeppelin is either local storage or s3, no GitHub integration but I wrote a python script in a notebook to handle that if we need
